ssofact.routing.yml

```diff
@@ -5,3 +5,12 @@ ssofact.redirect_login:
     _title: 'OpenID Connect ssoFACT login form redirect page'
   requirements:
     _custom_access: '\Drupal\ssofact\Controller\SsofactRedirectController::access'
+ssofact.user:
+  path: '/user/sso'
+  defaults:
+    _controller: '\Drupal\ssofact\Controller\Ssofact::content'
+    _title: 'ssoFACT'
+  requirements:
+    _permission: 'change own username'
+  options:
+    no_cache: true
```

Comment:
If this necessary for now, then we need to

1. add a YAML code comment to explain why it's needed and

1. inform NewsFactory accordingly that they need to cache the output for at least 1 hour.

If I understood the discussion correctly, then the asset cache from another page seems to be reused for the page /user/sso if the other page primed the cache upfront.  This most likely means that we need to change a cache key in the asset data, so that the asset bundles are not considered the same in terms of caching.

Reply:
I did not get, how to include the cache tags for the assets. Could you please give me a hand there?

Reply:
I'm not super familiar with that either right now, so let's postpone that issue to a later point in time.